loggingpy/log.py

```diff
@@ -142,10 +142,11 @@ def with_sink(sink):
         Logger.sinks.append(sink)
         sink.setFormatter(JsonFormatter())  # all appended sinks get the json formatter in order to log only structured messages
 
-    def __init__(self, context: str = ""):
+    def __init__(self, context: str = "", prefix_payload_type: bool = False):
```

Comment:
I would suggest to do it defensively and default this to true. Otherwise convince me otherwise :)

Reply:
Right, in the end, we decided to enable it by default. I will fix this.